app/src/pages/inside/dashboardItemPage/widgetWizardModal/widgetControls/controls/filtersControl/filtersWrapper/filtersWrapper.scss

```diff
@@ -0,0 +1,20 @@
+.filters {
+    &-wrapper {
+        width: calc(100% - 20px);
+        min-height: 18px;
+        margin: 1px 0 0;
+        padding: 10px 10px;
+        background: $COLOR--white;
+    }
+
+    &-text {
+        color: $COLOR--gray-47;
+        font-size: 12px;
+        font-weight: 600;
```

Comment:
Do not use font-weight property. We have global variables for all font types in `src/common/css/variables/font-variable.scss`